lib/cocina/models/builders/title_builder.rb

```diff
@@ -8,42 +8,44 @@ module Builders
       # TitleBuilder selects the prefered title from the cocina object for solr indexing
       class TitleBuilder # rubocop:disable Metrics/ClassLength
         extend Deprecation
-        # @param [[Array<Cocina::Models::Title,Cocina::Models::DescriptiveValue>] titles the titles to consider
+        # @param [Array<Cocina::Models::Title,Cocina::Models::DescriptiveValue>] titles the titles to consider
+        # @param [Array<Cocina::Models::FolioCatalogLink>] catalog_links the folio catalog links to check for digital serials part labels
         # @param [Symbol] strategy ":first" is the strategy for selection when primary or display
         #   title are missing
         # @param [Boolean] add_punctuation determines if the title should be formmated with punctuation
         # @return [String, Array] the title value for Solr - for :first strategy, a string; for :all strategy, an array
         #   (e.g. title displayed in blacklight search results vs boosting values for search result rankings)
-        def self.build(titles, strategy: :first, add_punctuation: true)
+        def self.build(titles, catalog_links: [], strategy: :first, add_punctuation: true)
```

Comment:
I like injecting the optional catalog links instead of tossing the whole Cocina at this method.